pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -210,6 +210,14 @@ func (r *ReconcileNetworkAddonsConfig) Reconcile(request reconcile.Request) (rec
 		return reconcile.Result{}, err
 	}
 
+	// Delete generated objsToRemove on Kubernetes API server
```

Comment:
Here you can drop the namespace from the list of objects (or in r.objectsToRemove, not sure what fits better). https://github.com/kubevirt/cluster-network-addons-operator/blob/39ca491bc9e99f98b996026adcd3f1fec8a2ffab/pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go#L46

Reply:
Sorry, to clarify, we should add a new var that reads from "OPERAND_NAMESPACE"

Reply:
done (on next push)